src/servers/exporter.ts

```diff
@@ -148,9 +150,14 @@ const main = async () => {
     res.end(await register.metrics())
   })
 
-  server.get("/healthz", async (req, res) => {
-    res.send("OK")
-  })
+  server.get(
+    "/healthz",
+    healthzHandler({
+      checkDbConnectionStatus: true,
+      checkRedisStatus: true, // TODO: set to false when invoices/payments updates are removed from get balance
```

Comment:
could you explain more in depth the comment?

Reply:
Previously we were using `getBalanceForWallet` (runs updatePendingInvoices and updatePendingPayments that need redis to lock wallets) but last week we updated to `getBalanceForWalletId` so now we no longer need redis check. Updated here https://github.com/GaloyMoney/galoy/pull/795/commits/cf8b2324ae2a1b56207a9d84a9b21eae8b71055a